From 9818473a6d6f255647c6e973573511401209199b Mon Sep 17 00:00:00 2001
From: Randy Dunlap <rdunlap@infradead.org>
Date: Tue, 30 Jun 2020 10:09:25 -0700
Subject: [PATCH 14/16] f2fs: always expose label 'next_page'
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Fix build error when F2FS_FS_COMPRESSION is not set/enabled.
This label is needed in either case.

../fs/f2fs/data.c: In function ‘f2fs_mpage_readpages’:
../fs/f2fs/data.c:2327:5: error: label ‘next_page’ used but not defined
     goto next_page;

Signed-off-by: Randy Dunlap <rdunlap@infradead.org>
Cc: Jaegeuk Kim <jaegeuk@kernel.org>
Cc: Chao Yu <yuchao0@huawei.com>
Cc: linux-f2fs-devel@lists.sourceforge.net
Link: https://lore.kernel.org/linux-f2fs-devel/020937f3-2947-ca41-c18a-026782216711@infradead.org/
Signed-off-by: Nathan Chancellor <natechancellor@gmail.com>
---
 fs/f2fs/data.c | 2 --
 1 file changed, 2 deletions(-)

diff --git a/fs/f2fs/data.c b/fs/f2fs/data.c
index 5cff2fdec65b..5fce352bcf00 100644
--- a/fs/f2fs/data.c
+++ b/fs/f2fs/data.c
@@ -2366,9 +2366,7 @@ static int f2fs_mpage_readpages(struct inode *inode,
 			zero_user_segment(page, 0, PAGE_SIZE);
 			unlock_page(page);
 		}
-#ifdef CONFIG_F2FS_FS_COMPRESSION
 next_page:
-#endif
 		if (rac)
 			put_page(page);
 
-- 
2.27.0

